config/manager/manager.yaml

```diff
@@ -67,6 +67,9 @@ spec:
           - --application-images-registry=docker.io/library
         image: controller:latest
         name: manager
+        env:
+        - name: SSM_DOCUMENT_NAME
+          value: "SageMaker-SpaceSSHSessionDocument"
```

Comment:
`SSM_` isn't generic. Maybe `AWS_SSM_DOCUMENT_NAME` ?

Medium term: would it be possible to set the default value to `WorkspaceSessionDocument`, as in not SageMaker specific?
Then you can override this value in the `makefile` / guided charts for SageMaker-specific use cases.